assets/translations/en.json

```diff
@@ -70,5 +70,8 @@
   "transactionTypeHere": "Type here ..",
   "transactionRecordIncome": "Record income",
   "transactionRecordExpenses": "Record expense",
+  "transactionUploadHere": "Upload here",
+  "transactionFileType": "File type : jpg, jpeg, png, pdf",
+  "transactionMaxSize": "Max size",
```

Comment:
Do you need to add "transaction" multiple times? why not 
 `{ "maxSize": "Max size" }`